app/controllers/vacancies_controller.rb

```diff
@@ -78,4 +80,10 @@ def trigger_dfe_analytics_event(event_type, event_data)
 
     DfE::Analytics::SendEvents.do([event])
   end
+
+  def set_search_coordinates
+    return unless form.to_hash[:location]
+
+    @search_coordinates = Geocoding.new(form.to_hash[:location]).coordinates
```

Comment:
This is already being cached in [geocoding#coordinates](https://github.com/DFE-Digital/teaching-vacancies/blob/8c020999187adf39a343bf21aac19dccb92ed304/lib/geocoding.rb#L14-L23).